week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,19 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  // b it.s not defined
   2. Remove the line that throws the error.
+  // The line 38 throws the error because we don't have a defined variable b.
   3. What is printed to the console?
+  // The console prints first the variabels x and a, then the numbers: 4,9,6,13,8.
   4. How many times is "f1" called?
+  //Three times
   5. How many times is "f2" called?
+  //Two times
```

Comment:
Are you sure about these? Could you write down for which values `f1` gets called and for which values `f2` gets called? Note that `f2` is used before `f1` in the loop, so it might be a bit confusing.